src/release/main.d

```diff
@@ -183,21 +183,21 @@ void main ( string[] params )
         }
     }
 
-    auto email_body = craftMail(con, repo, myrelease.type,
+    auto recipient = getConfig("neptune.mail-recipient").ifThrown("");
+
+    auto email = craftMail(con, repo, myrelease.type, recipient,
         list.actions
             .map!(a=>cast(ReleaseAction) a)
             .filter!(a=>a !is null)
             .array);
 
-    writefln("This is the announcement email:\n-----\n%s\n-----", email_body);
-
-    auto recp = getConfig("neptune.mail-recipient").ifThrown("");
+    writefln("This is the announcement email:\n-----\n%s\n-----", email);
 
-    if (recp.length == 0)
+    if (recipient.length == 0)
```

Comment:
Order of actions is weird. First you get recipient, then craft mail and only after that check if recipient  is non-zero.

Also turning exception into empty string to immediately check for empty string after is just exception anti-pattern. But that part was already present before.